CIP-0XXX/README.md

```diff
@@ -0,0 +1,253 @@
+---
+CIP: ?
+Title: Votes & Certificates on Cardano
+Category: ?
```

Comment:
```suggestion
Category: Tools
```
Please correct me if this assumption is wrong, but since this scheme is external to the Ledger and implemented with a broadly used library it would come under the `Tools` category.